pkg/agent/runner.go

```diff
@@ -89,50 +89,36 @@ type Runner struct {
 
 	// vm stores some common information about the VM.
 	//
-	// FIXME: the value of vm.Using() is sometimes inaccurate - we need to prematurely mark
-	// resources as being used so that a new scheduler won't send a Register request at the same
-	// time as we're updating the Informant/NeonVM with an increase that was already approved. This
-	// is also tied to handling migration; there's a comment specifically about migration that
-	// should handle this.
-	//
-	// This field MUST NOT be updated without holding BOTH lock AND vmStateLock.
-	//
-	// This field MAY be read while holding EITHER lock OR vmStateLock.
+	// This field MUST NOT be read or updated without holding lock.
```

Comment:
which lock? 

There's a pattern I got off of someone last year, where you put the mutex directly above all the fields that it's supposed to protect to indicate this.

Reply:
> which lock?

There's a field named `lock`... perhaps that name should be changed :sweat_smile:

> There's a pattern I got off of someone last year, where you put the mutex directly above all the fields that it's supposed to protect to indicate this.

Yeah, I'm definitely a fan of that approach as well. The `Runner` struct _in general_ is kind of a lot, and I don't think reorganizing the fields would help much.